src/types/index.ts

```diff
@@ -48,17 +51,8 @@ export interface Person {
   popularity: number;
 }
 
-export interface Collection {
-  id:number;
-  backdrop_path: string;
-  name: string;
-  poster_path: string;
-}
-
 export interface Movie {
   id: number;
-  logo_path: string;
```

Comment:
Did the api response change for this?

Reply:
I have tested multiple endpoints which respond with the Movie interface and none of these had the logo_path or name property.